src/Pages/Stay/StayMain/Calendar/StayCalendar.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import 'react-dates/initialize';
+import './datepicker.css';
+import { DateRangePicker } from 'react-dates';
+
+import styled from 'styled-components';
+
+const StayCalendar = ({ onCreate }) => {
+  const [startDate, setStartDate] = useState(null);
+  const [endDate, setEndDate] = useState(null);
+  const [startDate2, setStartDate2] = useState(null);
+  const [endDate2, setEndDate2] = useState(null);
+  const [startPlaceholder] = useState(new Date());
+  const [endPlaceholder] = useState(new Date());
+  const [focusedInput, setFocusInput] = useState(null);
+
+  const onDatesChange = ({ startDate, endDate }) => {
+    setStartDate(startDate);
+    setEndDate(endDate);
+
+    if (startDate !== null && endDate !== null) {
+      setStartDate2(startDate.format('YYYY-MM-DD'));
+      setEndDate2(endDate.format('YYYY-MM-DD'));
+    }
+
+    onCreate({
+      start: startDate2,
+      end: endDate2,
+    });
```

Comment:
startDate2, endDate2에 단 리뷰와 같은 내용으로, 아래와 같이 수정할 수 있지 않을까요?

```suggestion
    /* startDate2라는 state를 만들어 넘기지 말고 바로 format한 데이터를 넘긴다 */
    onCreate({
      start: startDate.format('YYYY-MM-DD'),
      end: /* 마찬가지 */,
    });
```

Reply:
이 부분은 부모의 setState를 props로 내려가지고와 format된 데이터를 바로 넘겨줘서 부모 state에 할당했습니다 감사합니다